.eslintrc.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
   },
   env: {
     node: true,
+    es6: true,
```

Comment:
I used a `Set()` in my solution, and this was needed for ESLint to understand the `Set` global.